table/queries/derived_satellite_scans.sql

```diff
@@ -1,52 +1,52 @@
 # https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-6
 CREATE TEMP FUNCTION ClassifySatelliteRCode(rcode INTEGER) AS (
   CASE
-    WHEN rcode = 1 THEN "❗️dns/rcode:FormErr"
-    WHEN rcode = 2 THEN "❗️dns/rcode:ServFail"
+    WHEN rcode = 1 THEN "⚠️dns/rcode:FormErr"
+    WHEN rcode = 2 THEN "⚠️dns/rcode:ServFail"
     WHEN rcode = 3 THEN "❗️dns/rcode:NXDomain"
```

Comment:
Why is the ServFail not as good an indicator of censorship as NXDomain is? Are we worried about geoblocking in the resolution process? Is there a specific example for this?

Reply:
These aren't set in stone. (Actually I think we're likely to change them a lot as we fine-tune) I'm just basing this on staring at the dashboards a lot for [this doc](https://docs.google.com/document/d/1OnfLgJ3OdghLiPf9tKdqgIl346EehXcZFLrcTtDKwE8/edit#) and noticing that lots of countries have some residue of servfail/refused that doesn't seem to be censorship.

Actually. we can quantify this. Looking at [this view](https://datastudio.google.com/s/v2sohsCeT9g) (just those three Rcode outcomes, fortunately all other codes are much less common)
![image](https://user-images.githubusercontent.com/1127209/199524801-cc2d4050-11eb-4e05-bdf7-784a3522bd9f.png)
ServFail appears in almost every country, while Refused and Nxdomain appear only in some countries.

Based on that I'm going to switch to
`⚠️dns/rcode:ServFail`
`❗️dns/rcode:NXDomain`
`❗️dns/rcode:Refused`